.env.example

```diff
@@ -5,3 +5,5 @@ SETTINGS__PROMETHEUS_EXPORTER_HOST=prometheus.compliance-test.svc
 APPLICATION_TYPE=test
 HOSTNAME=testmachine
 OPENSHIFT_BUILD_NAME=testmachine-test-42
+export MINIO_ACCESS_KEY=BQA2GEXO711FVBVXDWKM
+export MINIO_SECRET_KEY=uvgz3LCwWM3e400cDkQIH/y1Y4xgU4iV91CwFSPC
```

Comment:
Added an .env.example and removed the .env file from the repo. It's already part of the .gitignore.